app/models/pacbio/plate_creator.rb

```diff
@@ -4,6 +4,10 @@
 # load order issues on production
 require 'pacbio/pacbio'
 
+# This is currently only being used for the creation of Pacbio::Plate
```

Comment:
[Yes it can!](https://github.com/sanger/traction-service/pull/953/files#diff-17b8dce2e574f76fed5c099738337b7b2a0f94db8fe81d34185c3a6cd0a200dd)